docs/scos/dev/back-end-development/zed-ui-tables/create-and-configure-zed-tables.md

```diff
@@ -129,6 +129,16 @@ $config->setDefaultSortDirection(
     \Spryker\Zed\Gui\Communication\Table\TableConfiguration::SORT_DESC);
 ```
 
+### Configure strict search
+
+By default, strict search is disabled. However, you can enable it when setting up the table.
```

Comment:
Please describe why do we need strict search and where we can use it. Also that it's case sensitive search